app/src/main/java/de/langerhans/odintools/tools/DeviceUtils.kt

```diff
@@ -4,20 +4,20 @@ import de.langerhans.odintools.tools.DeviceType.*
 import javax.inject.Inject
 
 class DeviceUtils @Inject constructor(
-    private val shellExecutor: ShellExecutor,
+    private val executor: ShellExecutor,
 ) {
 
+    fun getDeviceVersion() = executor.getStringProperty(SettingsRepo.KEY_BUILD_VERSION, "")
+
+    fun getDeviceVendor() = executor.getStringProperty(SettingsRepo.KEY_VENDOR_NAME, "")
```

Comment:
Technically this value is not the vendor. Suggestion: `getDeviceCodename()`

Reply:
Sounds good!